python/ray/worker.py

```diff
@@ -994,6 +993,11 @@ def _wait_for_and_process_task(self, task):
                 next_title = "ray_{}".format(actor.__class__.__name__)
             with profiling.profile("task", extra_data=extra_data):
                 with _changeproctitle(title, next_title):
+                    print("Ray driver id: {}".format(str(driver_id)))
```

Comment:
This is being printed for every task, right? Can we print it only when the driver ID changes? Otherwise the log files will grow in proportion to the number of tasks.

Reply:
Completely missed that, thanks for the catch

Reply:
I just tried the latest version, and this is still getting printed once per task for me.